src/components/Navigation.jsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { HumanLoggedIn } from './Icons';
 
 function Navigation() {
   return (
-    <header>
-      <h1>Bookstore</h1>
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/categories">Categories</Link>
-          </li>
-        </ul>
-      </nav>
-    </header>
+    <nav>
```

Comment:
- [ ] Certainly! You have a keen eye for detail. If you observe closely, you'll notice that the layout of the navbar on the left side aligns perfectly with the card, just as intended in the design. However, the icon on the right side isn't in the correct position as specified in the design. To address this, I kindly recommend following the design guidelines closely. Update the layout of the icon on the right side so that it horizontally aligns with the card. By making this adjustment, you'll ensure a harmonious and consistent alignment throughout your design.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/AliRezaBmeDu/bookstore/assets/55840999/610411ef-6cd8-48ce-8422-4af092422bfc) | ![image](https://github.com/AliRezaBmeDu/bookstore/assets/55840999/27862459-2376-4344-9410-eeff361f7084) |

Your dedication to aligning these elements according to the design will enhance the overall visual appeal of your project. Keep up the great work! 🎉📐